.circleci/config.yml

```diff
@@ -543,87 +543,12 @@ workflows:
     jobs:
       - prepare-system
 
-      - prepare-python-linux:
```

Comment:
You will need to also change the following:

- README.rst
- The docker files
- tox.ini
- The setup guide at contributing.md
- The setup guide at macos_install_guide.rst
- The setup guide at overview_and_guide.rst